test/utils/navigation.spec.ts

```diff
@@ -172,7 +172,7 @@ describe("NAVIGATION utils", () => {
     expect(navigation).toEqual(config.OVERSEAS_ENTITY_QUERY_URL);
   });
 
-  test(`NAVIGATION returns ${config.UPDATE_INTERRUPT_CARD_URL} when calling previousPage on ${config.OVERSEAS_ENTITY_QUERY_URL} object`, () => {
+  test.skip(`NAVIGATION returns ${config.UPDATE_INTERRUPT_CARD_URL} when calling previousPage on ${config.OVERSEAS_ENTITY_QUERY_URL} object`, () => {
```

Comment:
skip will need removing

Reply:
Yep, as per comment in the commit message. I've got a commit ready to go now, with this test fixed.